### ISSUE TITLE
Pipeline fails on process `multiqc`: input file name collision

### ISSUE THREAD

**Issue by dfornika** [BCCDC-PHL/routine-sequence-qc#41]
We're seeing some pipeline failures with the following error message:

```
Error executing process > 'multiqc (<run_id>)'

Caused by:
  Process `multiqc` input file name collision -- There are multiple input files for each of the following file names: sample-01_Species_multiqc_bracken.txt, sample-02_Species_multiqc_bracken.txt...

Tip: when you have fixed the problem you can continue the execution adding the option `-resume` to the run command line
Unexpected error [NullPointerException]
Unexpected error [NullPointerException]
```

**Comment by nimbusFarzaneh** [comment]
Is this for run 210907_VH00278_49_AAALVGKM5?

**Comment by dfornika** [comment]
Unfortunately this issue was not actually fixed in #43
